src/client/src/orangehrmRecruitmentPlugin/pages/ApplyJobVacancy.vue

```diff
@@ -259,14 +259,14 @@ export default {
     },
   },
   setup() {
-    const defaultPic = `${window.appGlobal.baseUrl}/../images/ohrm_branding.png`;
+    const defaultPic = `${window.appGlobal.publicPath}/images/ohrm_branding.png`;
```

Comment:
@Super-Chama I changed the way of accessing static assets, basically changed the base path